### PR TITLE
feat(rcollate)

### PR DESCRIPTION
**Pull request by built-tools**
Added rcollate and tests.

### CODE REVIEW

File: honeybee_radiance_command/options/rcollate.py [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+class RcollateOptions(OptionCollection):
+    """Rcollate command options.
+
+    [ -h[io] ][ -w ][ -f[afdb][N] ][ -t ][ -ic in_col ][ -ir in_row ][ -oc out_col ]
+    [ -or out_row ][ -o RxC[xR1xC2..] ]
+
+    In this class the -f option has been implemented such that there are separate options
+    like -fa , -ff etc. This is to enable the inclusion of numeric inputs for digits
+    in an easy manner.
```
Comment (mostaphaRoudsari) [inline honeybee_radiance_command/options/rcollate.py:15]: Really like this idea!

File: honeybee_radiance_command/options/rcollate.py [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+                                            'format.')
+        self._fb = StringOptionJoined('fb', 'Number of words in the input,'
+                                            ' each record assumed to be in binary '
+                                            'format.')
+        self._t = BoolOption('t', 'Swap rows and columns in the input.')
+        self._ic = IntegerOption('ic', 'Input columns')
+        self._ir = IntegerOption('ir', 'Input rows')
+        self._oc = IntegerOption('oc', 'Output columns')
+        # or  is a protected keyword, so using oR
+        self._oR = IntegerOption('or', 'Output rows')
```
Comment (mostaphaRoudsari) [inline honeybee_radiance_command/options/rcollate.py:45]: We had cases like this before and we used `_` in those cases. We might want to do the same here to be consistent. Here is one for `as`:

https://github.com/ladybug-tools/honeybee-radiance-command/blob/e8f7459a6d36f6cd4a9df2ffbfd5ee9441188517/honeybee_radiance_command/options/rtrace.py#L709-L720

File: tests/options/rcollate_test.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+
+def test_from_string():
+    opt = RcollateOptions()
+    opt_str = '-h -oc 8760 -fa3'
+
+    opt.update_from_string(opt_str)
+    assert opt.h == ''
+    assert opt.oc == 8760
+    assert opt.fa == '3'
```
Comment (mostaphaRoudsari) [inline tests/options/rcollate_test.py:47]: This is good for now but we should start writing tests for failures too. For instance a test for the `-o` option you want to make sure it raises the correct errors for invalid inputs.

**Review by mostaphaRoudsari — APPROVED** [review]
LGTM! I left a few comments for possible improvements.

**Comment by built-tools** [comment]
@mostaphaRoudsari Okay to all. Regarding tests and optimization I have some ideas to improve both, however, I think for the time being, I am just trying to keep the progress moving so that we don’t get stuck with this repo alone.

**Comment by built-tools** [comment]
@mostaphaRoudsari The changes have been made as per review.

**Review by mostaphaRoudsari — APPROVED** [review]
Hi @built-tools! LGTM. Thanks.

**Pull request merged**